reactioncommerce/reaction
# Update NPM packages

Impact: **minor**  
Type: **feature**

Updates some NPM dependencies with low to moderate vulnerabilities. Primarily Jest and other dev dependencies.

## Breaking changes
None

## Testing
1. Clone branch and npm install
2. Verify that `npx snyk test` reports no vulnerabilities.
3. Verify that `npm audit` reports no vulnerabilities.
4. Verify that all tests pass on CI
5. Verify that the app starts up, basic smoke test, etc.
